index.js

```diff
@@ -116,7 +117,7 @@ module.exports = function (babel) {
               let msg = `Only \`import hbs from '${matchingModulePath}'\` is supported. You used: \`${usedImportStatement}\``;
               throw path.buildCodeFrameError(msg);
             }
-          } else {
+          } else if (modulePathExport !== 'hbs') {
```

Comment:
Why is this change required? `ember-cli-htmlbars` already configures the `modules` option for us (and it works just fine without this `else if` change).

Reply:
This line change allows for errors to be thrown if user/developer writes:

```js
import hbs from 'ember-cli-htmlbars';
```

while it should be:

```js
import { hbs } from 'ember-cli-htmlbars';
```

We might be able to find a different way to accomplish this. I was kind of satisfied with a single line change considering it passed both versions of tests and performance implications would be minimal(single if check). I'm of course willing to change it if we can find a better way :)

Reply:
The issue is that we shouldn't hard code the "hbs" name here.

Reply:
@rwjblue updated the the code, tests passing now. Also did few manual tests just to be sure :sweat_smile: